spec/unit/util/monkey_patches_spec.rb

```diff
@@ -106,7 +106,7 @@ def with_root_certs(certs)
       with_root_certs([cert])
       store.add_cert(cert)
 
-      expect(store).to receive(:warn).with('Failed to add /DC=com/DC=microsoft/CN=Microsoft Root Certificate Authority')
+      expect(store).to receive(:warn).with('Failed to add CN=Microsoft Root Certificate Authority,DC=microsoft,DC=com')
```

Comment:
Note if the distinguished name has multiple components, then the order is reversed from what it was prior. I think the new behavior is more correct/expected.